src/Compilers/VisualBasic/Portable/Symbols/Source/SourceParameterSymbol.vb

```diff
@@ -322,6 +332,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 arguments.GetOrCreateData(Of CommonParameterWellKnownAttributeData)().HasOutAttribute = True
             ElseIf attrData.IsTargetAttribute(Me, AttributeDescription.MarshalAsAttribute) Then
                 MarshalAsAttributeDecoder(Of CommonParameterWellKnownAttributeData, AttributeSyntax, VisualBasicAttributeData, AttributeLocation).Decode(arguments, AttributeTargets.Parameter, MessageProvider.Instance)
+            ElseIf attrData.IsTargetAttribute(Me, AttributeDescription.CallerArgumentExpressionAttribute) Then
+                Dim index = GetEarlyDecodedWellKnownAttributeData()?.CallerArgumentExpressionParameterIndex
```

Comment:
>GetEarlyDecodedWellKnownAttributeData()?.CallerArgumentExpressionParameterIndex [](http://example.com/codeflow?start=28&length=79)

A conditional access impies a possibilty of a null value. Nothing is going to be reported for it. Is this intentional or we don't expec a null value?

Reply:
Other callers in the same code path does null checks. I'm not able to find a scenario where it could be null. But I think it doesn't harm to keep the check given that other callers do that?